src/libs/Navigation/Navigation.js

```diff
@@ -321,19 +204,15 @@ export default {
     isActiveRoute,
     getActiveRoute,
     goBack,
-    closeDrawer,
-    getDefaultDrawerState,
-    setDidTapNotification,
     isNavigationReady,
     setIsNavigationReady,
     getReportIDFromRoute,
-    isDrawerReady,
-    setIsDrawerReady,
-    resetDrawerIsReadyPromise,
     resetIsReportScreenReadyPromise,
-    isDrawerRoute,
     isReportScreenReady,
     setIsReportScreenIsReady,
+
+    // Re-exporting the getTopmostReportId here to fill in default value for state. The getTopmostReportId isn't defined in this file to avoid cyclic dependencies.
+    getTopmostReportId: (state = navigationRef.getState()) => getTopmostReportId(state),
```

Comment:
Do you mean to use the default value in the place where this function is defined `getTopmostReportId.js`?  The `navigationRef` is defined here so it would create a circular dependency